bootstrap/bootstrap.ps1

```diff
@@ -52,9 +56,12 @@ if ($LastExitCode -ne 0)
 
 # create a junction to the shared FX version directory. this is
 # so we have a stable path to dotnet.exe regardless of version.
-$junctionTarget = Join-Path $toolsLocalPath "dotnetcli\shared\Microsoft.NETCore.App\1.0.0"
```

Comment:
I would prefer this stay in here instead of duplicating this in each repo. I also don't like the hardcoding of the 1.0.0 shared framework here or in the individual repos. I like your other suggestion of finding the latest version that installed with the cli and linking to that.